tests/org/test_add_volunteer_api.py

```diff
@@ -0,0 +1,123 @@
+from org.models import *
+from tests.AuthAPITestCase import AuthAPITestCase
+from rest_framework.test import APITestCase,APIClient
+from rest_framework import status
+from users.models import User
+from org.custom_model_field import PermissionSet as Permissions
+import uuid
+class AddVolunteerAPITestCase(AuthAPITestCase):
+    """
+    This class is to test the API [get] /api/org/[org-id]/volunteer
+    present in the view Org.views.AddVolunteer.get
+    """
+    def setUp(self):
+        """
+            Create an organization in the test database
+        """
+        # Create the org
+        self.org = Org.objects.create(
+            route_slug= 'route_slug',
+            name='test',
+            tagline='test',
+        )
+
+        # Create default group permissions  
+        self.volunteer_permissions = Permissions()
+        
+        #Create volunteer permission set objecct
+        self.volunteer_permission_set = PermissionSet.objects.create(
+            name = 'Volunteer',
+            org = self.org,
+            permissions = self.volunteer_permissions 
+        )
+
+
+        self.volunteer_group_invite_slug = str(self.org.id)+'-'+str(uuid.uuid4())
+
+        #Create volunteer group with organization
+        self.volunteer_group = Group.objects.create(
+            name='Volunteer',
+            role='''When a person clicks join org button of his own without a 
+                invite link then he will be put into this group.''',
+            invite_slug=self.volunteer_group_invite_slug,
+            org=self.org,
+            default_permission_set=self.volunteer_permission_set
+        )
+        
+        #Create auth user object
+        self.user_email = 'test@host.com'
+        self.user_password = 'test_user.123'
+        self.user = User.objects.create_user(
+            email=self.user_email,
+            username=self.user_email,
+            name="test user",
+            password=self.user_password,
+            institution='E-Cell',
+            country_code='+91',
+            phone='5446465452'
+        )
+    
+    def create_auth_client(self):
```

Comment:
By using `tests.AuthAPITestCase.AuthAPITestCase` we can avoid this code duplication. 

Ex. `tests.org. CreateOrgAPITestCase`